packages/core/src/video/VideoForRendering.tsx

```diff
@@ -110,7 +119,7 @@ const VideoForRenderingForwardFunction: React.ForwardRefRenderFunction<
 			return getMediaTime({
 				fps: videoConfig.fps,
 				frame,
-				src: props.src as string,
+				src: videoRef.current.currentSrc,
```

Comment:
Does it still work if the source gets changed?

We could test it by rendering 1 second with 1 video, then rendering 1 second with another video.

It's raising my eyebrow because src is not in the useEffect dependencies anymore.